### PR TITLE
Update Ledger API version and its documentation

### PR DESCRIPTION
**Pull request by mziolekda**
CHANGELOG_BEGIN
Ledger API version bump to 2.1
CHANGELOG_END

### CODE REVIEW

**Review by skisel-da — APPROVED** [review]

File: docs/source/support/compatibility.rst [hunk 1]
```diff
@@ -66,6 +68,12 @@ Summary of Ledger API Changes
 
    * - Ledger API Version
      - Changes
+   * - 2.1
+     - | Establish the order of child events in ExercisedEvent to agree with the order of events in transaction.
+       | Indicate an exercise done on an interface through the interface_id field on the ExercisedEvent message.
+       | Make interfaces available for subscriptions in the Transaction Service as a preview feature.
```
Comment (bame-da) [inline docs/source/support/compatibility.rst:74]:
```suggestion
       | Make interfaces available for subscriptions in the Transaction Service as an Alpha feature.
```

File: docs/source/support/compatibility.rst [hunk 2]
```diff
@@ -66,6 +68,12 @@ Summary of Ledger API Changes
...
+       | Implement contract disclosure as an experimental feature in the Transaction, Command Submission and Command Services. Related new message types of DisclosedContract and ContractMetadata should be ignored.
```
Comment (bame-da) [inline docs/source/support/compatibility.rst:75]:
```suggestion
       | Implement contract disclosure as a Labs feature in the Transaction, Command Submission and Command Services. Related new message types of DisclosedContract and ContractMetadata should be ignored.
```

**Review by bame-da — APPROVED** [review]
Approved with the suggestions. Thank you!

File: docs/source/support/compatibility.rst (hunk 1 above)
Comment (mziolekda) [reply docs/source/support/compatibility.rst]: :+1:

File: docs/source/support/compatibility.rst (hunk 2 above)
Comment (mziolekda) [reply docs/source/support/compatibility.rst]: :+1:

**Pull request merged**